extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1733,11 +1750,11 @@ static public function getGroups() {
 	 */
 	static public function getVariables( $sort = "cv_name", $wiki = 0, $group = 0,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "cv_name" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)